SparkBuilderGeneratorPlugin/src/com/helospark/spark/builder/handlers/codegenerator/ApplicableBuilderFieldExtractor.java

```diff
@@ -47,7 +45,7 @@ public ApplicableBuilderFieldExtractor(ClassFieldCollector classFieldCollector,
 
     private boolean fieldDeclarationsContainConstructorField(List<? extends BuilderField> applicableFields, BuilderField field) {
         for (BuilderField currentField : applicableFields) {
-            if (currentField instanceof ConstructorParameterSetterBuilderField && currentField.getBuilderFieldName().equals(field.getBuilderFieldName())) {
+            if (currentField.getBuilderFieldName().equals(field.getBuilderFieldName())) {
```

Comment:
Just a simple return?

Reply:
We cannot do that. We have to return true, if we found one that matches, but we only return false if none matched.